### ISSUE TITLE
The default value of the region option is us-east-1

### ISSUE THREAD

**Issue by t4kamura** [go-to-k/cls3#51]
I am using cls3 version `0.5.1`.

The `README.md` says that the default value of the region option is `ap-northeast-1`, but the actual default value seems to be `us-east-1`.

https://github.com/go-to-k/cls3/blob/29ab161874a3223e2b3fa6fbfc6ebda7d139a5e7/README.md?plain=1#L35-L36

https://github.com/go-to-k/cls3/blob/29ab161874a3223e2b3fa6fbfc6ebda7d139a5e7/aws_config.go#L10

When I tried to delete an S3 bucket in `ap-northeast-1` without specifying region, I got an error.

```
ERR PermanentRedirectError: Are you sure you are specifying the correct region?
```

Which is the default value you want I don't know, but I am reporting it because the `README.md` and the actual value seem to be different.
[The blog post](https://go-to-k.hatenablog.com/entry/cls3) is also `ap-northeast-1`, so is `ap-northeast-1` correct?

**Comment by go-to-k** [comment]
@t4kamura 

Hi, Thanks for your reports.

It was a complete blind spot... As documented, I fixed it to ap-northeast-1 in v0.5.2 and released it!
Please use it again if you like!

https://github.com/go-to-k/cls3/releases/tag/v0.5.2

**Comment by t4kamura** [comment]
@go-to-k 
Thank you for your swift response!
I have and will continue to use it!
